docs/dev-env-first-time-contributors.md

```diff
@@ -701,54 +701,10 @@ patching file bundler.rb
 
 See ["Possible testing issues"](testing.html#possible-testing-issues).
 
-### `provision.py` fails during `vagrant up` on Ubuntu
 
-If you see the following error after running  `vagrant up` :
+### `pip install` fails during `vagrant up` on Ubuntu
```

Comment:
I like the headline of this section; the person doing the installation has just run the `vagrant up` step, so I think it's a good idea to use that word prominently in the section title, so people see it when they're skimming. And `pip install` is the particular bit of that process that's failing. Good summary!